### PR TITLE
Compare and Objects

### PR DESCRIPTION
**Pull request by lukaszspyra**
Task explained with comments in main() and tests to present comparing objects using helper class Objects.

### CODE REVIEW

**Comment by marcin-ogorzalek-EPAM** [comment]
Please increase the project's version in the `pom.xml`

File: pom.xml [hunk 1]
```diff
@@ -14,7 +14,9 @@
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-        <project.java.version>11</project.java.version>
+        <project.java.version>14</project.java.version>
+        <maven.compiler.source>${project.java.version}</maven.compiler.source>
+        <maven.compiler.target>${project.java.version}</maven.compiler.target>
```
Comment (marcin-ogorzalek-EPAM) [inline pom.xml:19]: Why those two linnes when there is `<maven.compiler.release>${project.java.version}</maven.compiler.release>`

File: pom.xml [hunk 2]
```diff
@@ -14,7 +14,9 @@
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-        <project.java.version>11</project.java.version>
+        <project.java.version>14</project.java.version>
```
Comment (marcin-ogorzalek-EPAM) [inline pom.xml:17]: While we are using a preview and JDK never than 11 I would suggest the newest (not early access one) version. That means 15.

Comment (lukaszspyra) [reply pom.xml]: Thank you, changed to 15.

File: pom.xml (hunk 1 above)
Comment (lukaszspyra) [reply pom.xml]: Unnecessary lines removed.

File: pom.xml [hunk 3]
```diff
@@ -6,15 +6,15 @@
 
     <groupId>com.epam.prejap.ess</groupId>
     <artifactId>helpers</artifactId>
-    <version>0.1</version>
+    <version>0.4</version>
```
Comment (marcin-ogorzalek-EPAM) [inline pom.xml:9]: Please bump up versions once more.

File: src/main/java/com/epam/prejap/ess/helpers/objectscompare/TaskObjectsCompare.java [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+ */
+final class TaskObjectsCompare {
+
+    private static PrintStream out = System.out;
+
+    public static void main(String[] args) {
+        Person lukasz = new Person(1, "Lukasz");
+        Person anthony = new Person(2, "Anthony");
+        Person david = new Person(3, "David");
+        List<Person> people = Arrays.asList(lukasz, david, anthony);
```
Comment (marcin-ogorzalek-EPAM) [inline src/main/java/com/epam/prejap/ess/helpers/objectscompare/TaskObjectsCompare.java:18]: Why not `List.of()` factory method?

File: pom.xml (hunk 3 above)
Comment (lukaszspyra) [reply pom.xml]: Ok, changed to 0.5

**Comment by marcin-ogorzalek-EPAM** [comment]
We have a merge conflict here, please resolve.

**Comment by lukaszspyra** [comment]
@marcin-ogorzalek-EPAM conflicts resolved :)

**Pull request merged**